edx_proctoring/api.py

```diff
@@ -881,18 +878,22 @@ def is_state_transition_legal(from_status, to_status):
 
 
 # pylint: disable=inconsistent-return-statements
-def update_attempt_status(exam_id, user_id, to_status,
+def update_attempt_status(attempt_id, to_status,
                           raise_if_not_found=True, cascade_effects=True, timeout_timestamp=None,
                           update_attributable_to=None):
     """
     Internal helper to handle state transitions of attempt status
     """
-    exam_attempt_obj = ProctoredExamStudentAttempt.objects.get_current_exam_attempt(exam_id, user_id)
```

Comment:
This is the line that was causing only the most recent attempt to be updated. I changed the arguments to this function so that it only takes in attempt id (which is unique), as opposed to an exam id and user id (which creates a combination that is not unique)